src/en/space-station-14/round-flow/proposals/specter.md

```diff
@@ -0,0 +1,151 @@
+Fear starts decaying naturally over time a little bit after they have stopped accumulating it.
+However, this decay increases with the level of the fear, so high fear levels decay quicker before slowing down.
+
+There are several levels of severity, each having their own effects:
+
+- **Normal:** The default state: No alert and no effect.
+- **Uneasy:** No effect. Serves as a warning to the player.
+- **Scared:** -25% melee damage to fear-inducing sources. 
+- **Terrified:** -50% melee damage to fear-inducing sources, 5% movement speed increase, and 10% increased damage from fear-inducing sources.
+- **Hysterical:** Inability to use weapons, 10% movement speed increase, and 25% increased damage from fear-inducing sources, susceptible to Specter's "Reap".
+
+Fear can be actively countered by consuming some chemicals.
+Common ones, like alcohol or nicotine, work only when Uneasy or Scared.
+However, more advanced calming chemical work even when a player is Hysterical.
+Sleeping can also be used to quickly drain built up fear.
+
+Certain jobs, like the Psychologist and Chaplain, start with a small amount of general fear-resistance.
+
+However, fear can also be made more potent by ingesting certain chemicals or having low health and stamina.
+
+## The Specter
+
+The Specter is a solo antagonist with the physical properties of a ghost.
+It is typically invisible to living players, can fly through walls around the station, and is generally impervious to any kind of attack.
+They can see both players and ghosts but are unable to use any form of conventional chat to communicate with them.
+However, they are able to read both normal and deadchat.
+
+They are able to see the fear level of player's through status icons.
+The terrified and hysterical fear levels are marked with a slow and fast flashing respectively, in order to signal when objectives can be completed or souls can be claimed.
+
+There are two primary resources that the Specter must manage: _power_ and _souls_.
+
+_Power_ is the Specter's resource pool for a few different things.
+Power is used both for activating abilities and as the Specter's health.
+
+_Terror_ is used by the Specter to unlock abilities that are already revealed.
+It is accumulated alongside Power when the Specter is nearby players with high amounts of fear.
+Using abilities causes the specter to be briefly visible but still unable to take damage.
+However, this allows people to throw salt or use the bible on them.
+
+_Souls_ are the Specter's form of progression.
+A soul is gained every time a _haunt_ is successfully completed.
+At 3, 6, and 9 souls, the Specter gains more powerful abilities that can be unlocked.
+
+The Specter is primarily counted through the use of holy items: salt and the Bible.
+Stepping on salt forces the Specter into the "haunt" mode, making them vulnerable.
+Using the bible on the Specter after it uses an ability causes the same effect.
```

Comment:
Should this maybe instead caused by if it "receives" holy damage (even if it also negates this to zero), meaning it would also be vulnerable to all other sources of holy damage (holy water, and the monk sticks)? This was kind of an extended topic when we added holy damage, that the Chaplain should not be the exclusive source of it, in case we ever add a gamemode that is more reliant on it. Making it bible only would kinda walk this back, although I guess there is also salt. Still, keeping this consistent with a single system would probably make sense? ((consider, adding holy (or a separate new type within metaphysical) damage to salt?))